Updated readme file (#26488)
demisto/content: Packs/CommonScripts/Scripts/ParseEmailFilesV2/README.md

| Email.AttachmentsData.FilePath | the location of the attachment, on the XSOAR server | String |
| Email.AttachmentNames | The list of attachment names in the email | string |
| Email.Format | The format of the email if available | string |

#### Notes

> Be aware, we handle EML and MSG parsing when the email contains HTML differently, if it's an EML and it has the ***content-type*** of ***text/html*** the content of the body will be stored in the html field.

Reviewer: ```suggestion
Be aware, we handle EML and MSG parsing differently when the email contains HTML. If it's an EML and it has the **content-type** of *text/html*, the content of the body will be stored in the html field.
```

> In case it's an MSG we will store the text inside the HTML in the text field and the HTML in the html field.

Reviewer: ```suggestion
If it's an MSG, we store the text inside the HTML in the text field and the HTML in the html field.
```
